src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -749,17 +1117,34 @@ describe('LibraryServiceClient', function() {
       var expectedResponse = {};
 
       // Mock Grpc layer
-      client._addLabel = function(actualRequest, options, callback) {
-        assert.deepStrictEqual(actualRequest, request);
-        callback(null, expectedResponse);
-      };
+      client._addLabel = mockSimpleGrpcMethod(request, expectedResponse);
 
       client.addLabel(request, function(err, response) {
         assert.ifError(err);
         assert.deepStrictEqual(response, expectedResponse);
         done();
       });
     });
+
+    it('invokes addLabel with error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var formattedResource = client.bookPath("[SHELF_ID]", "[BOOK_ID]");
+      var label = 'label102727412';
+      var request = {
+          resource : formattedResource,
+          label : label
+      };
+
+      // Mock Grpc layer
+      client._addLabel = mockSimpleGrpcMethod(request, null, error);
+
+      client.addLabel(request, function(err, response) {
+        assert(err instanceof Error);
+        assert.equal(err.code, FAKE_STATUS_CODE);
+        done();
+      });
+    });
   });
 
   describe('getBigBookAsync', function() {
```

Comment:
What is causing this to be `getBigBookAsync`. Shouldn't it be [`getBigBook`](https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline#L1782)?

Furthermore, we need to do something different for longrunning methods since they will be wrapped with this [wrapper](https://github.com/googleapis/gax-nodejs/blob/master/lib/longrunning.js#L134).